### PR TITLE
add desc for system events [fixes #32748]

### LINKED ISSUE #32748: Missing Extended Description for system events
Reproduce issue [#2025](https://github.com/docker/docker.github.io/issues/2025)  from docker/docker.github.io

### PR DESCRIPTION
**Pull request by michaelspets**
**- What I did**
add desc for system events command

**- How I did it**
add system_events.md file to docs

Fixes #32748

### CODE REVIEW

File: docs/reference/commandline/system_events.md
Comment (mstanleyjones) [inline docs/reference/commandline/system_events.md]: Add a level two header Examples here

**Review by mdlinville — APPROVED** [review]
LGTM!

File: docs/reference/commandline/system_events.md
Comment (mstanleyjones) [inline docs/reference/commandline/system_events.md]: Disregard. Can't delete this from mobile.

Comment (mstanleyjones) [inline docs/reference/commandline/system_events.md]: Sory gut can you please undo this change? When I wrote the comment, I didn't realize that you had an Examples header below.

Also, for this repository, you need to squash your commits down into one before final review. To do that, issue the command `git rebase -i master`. An editor (probably vi) will open, change the lines for all the commits after the first one to say `squash` instead of `pick`, then save. You will get a chance to edit the combined commit messages. Make sure you don't remove the `Signed-off by` line. Save that editor, then you will end up with only one commit. Then to get it into the pull request, you need to do a force push `git push -f`.

**Review by mdlinville — CHANGES_REQUESTED** [review]

**Review by mdlinville — APPROVED** [review]

File: docs/reference/commandline/system_events.md
Comment (allencloud) [inline docs/reference/commandline/system_events.md]: I am afraid that type is also supporting plugin.
```
root@ip-172-31-23-42:~# docker events --filter type=plugin
2017-04-23T17:54:22.779942297Z plugin disable 07b3bcea70bffa4512000861684e634b8cc6675e1feb6c009c2629092f729355 (name=07b3bcea70bf)
```

With version info:
```
root@ip-172-31-23-42:~# docker version
Client:
 Version:      17.04.0-ce
 API version:  1.28
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 18:01:08 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.04.0-ce
 API version:  1.28 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   4845c56
 Built:        Mon Apr  3 18:01:08 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

Comment (michaelspets) [inline docs/reference/commandline/system_events.md]: Type is mentioned below in the doc as an experimental feature and I think that this is the reason why it is not with the othres supporting pluggins... Is it still an experimental??? and finally do you instist on the above changes??

Comment (vdemeester) [inline docs/reference/commandline/system_events.md]: `plugin` are no more experimental so they should be there 👼

**Review by vdemeester — APPROVED** [review]
LGTM 🐸 
/cc @thaJeztah what did we do for aliased commands ? (like `run` vs `container run`)

**Comment by thaJeztah** [comment]
> what did we do for aliased commands ? (like run vs container run)

good question; I _think_ we should redirect one to the other, otherwise it's very easy to have these docs get out of sync

**Comment by thaJeztah** [comment]
But /cc @johndmulhausen for input

**Comment by johndmulhausen** [comment]
We don't do anything with aliases that this PR needs to worry about, because currently we aren't surfacing that at all. For example: https://docs.docker.com/engine/reference/commandline/image_rm/

No info on aliases. You can see that aliases are listed: https://github.com/docker/docker.github.io/blob/master/_data/engine-cli/docker_image_rm.yaml

I don't think the alias information we have in the YAML is high quality enough to change anything at the moment. "rmi, remove" (in this example) actually resolves to "docker rmi" and "docker image remove" -- but how would we know that? 

At any rate, this shouldn't hold up the PR.

**Comment by thaJeztah** [comment]
Ok, let me merge this for now, but we should have a look at it, otherwise we end up maintaining two (possibly more) files with exactly the same content

**Pull request merged**
